statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/api/response/BreakdownResult.java

```diff
@@ -17,7 +19,11 @@
 
   public BreakdownResult(FacetValue breakdownBy, List<StatisticsResult> results) {
     this.breakdownBy = breakdownBy;
-    this.results = new ArrayList<>(results);
+    // By ordering first and reversing after, we ensure that the highest value is the first element
+    this.results = new ArrayList<>(results).stream()
```

Comment:
This looks fine for the constructor but what about the setter method?
Shouldn't we be consistent with that, in other words, shouldn't we expect the same behavior on the setter and constructor?